core/trino-main/src/main/java/io/trino/testing/MaterializedResult.java

```diff
@@ -253,13 +253,13 @@ else if (BIGINT.equals(type)) {
             type.writeLong(blockBuilder, (Long) value);
         }
         else if (INTEGER.equals(type)) {
-            type.writeLong(blockBuilder, (Integer) value);
+            type.writeLong(blockBuilder, ((Integer) value).longValue());
```

Comment:
```suggestion
            type.writeLong(blockBuilder, (Integer) value);
```

-- `.longValue()` is redundant here

(same below)